srcts/src/shiny/bind.ts

```diff
@@ -279,8 +306,6 @@ async function bindOutputs(
       // Check if ID is falsy
       if (!id) continue;
 
-      bindingsRegistery.addBinding(id, "output");
-
```

Comment:
There are a few conditions in the lines below (not seen in the diff) where we would `continue` and not bind to `el` because it's already bound. If we add this `id` to the `bindingsRegistry` here, then we'll overcount the bindings on `id`.